### DISCUSSION TITLE
how I can draw a control or another named widget on the screen?

### DISCUSSION THREAD

**Discussion by burhansvural** [flet-dev/flet#2555] [Q&A] [answered]
### Question

Is there anyone who can explain, give examples and explain how I can draw a control or another named widget on the screen?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
What do you mean by "draw"?

**Reply by burhansvural** [reply]
![question_112](https://github.com/flet-dev/flet/assets/149883930/791f5fac-3ee3-401d-9fb1-4230d9f5fe80)
@ndonkoHenri 
What do you mean by "draw"?
example source:
https://www.linkedin.com/pulse/getting-started-low-level-graphics-programming-embedded-soheil-nazari/

**Reply by ndonkoHenri** [reply, marked as answer]
You could make use of chart controls. See [here](https://flet.dev/search?q=chart).

**Reply by burhansvural** [reply]
@ndonkoHenri thanks.
